packages/bridge/contracts/l2/L2DaiGateway.sol

```diff
@@ -54,10 +51,30 @@ contract L2DaiGateway is L2ArbitrumGateway {
 
   address public immutable l1Dai;
   address public immutable l2Dai;
+  address public immutable l1Counterpart;
+  address public immutable l2Router;
   uint256 public isOpen = 1;
 
+  uint256 public exitNum;
```

Comment:
If you are not implementing tradeable exits on the L1 side, there is no need to keep track of an `exitNum`.

Reply:
If you remove it, I think it would be good to keep the same event signature in `WithdrawalInitiated` and the L1 counterpart.